.gitignore

```diff
@@ -15,3 +15,5 @@ kubeconfig_*
 *.tfplan
 
 cluster-keys.json
+examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/claim/microservice-claim.yaml
+.gitignore
```

Comment:
Why do we want to ignore these?

Reply:
I've added claim to the ignore as readme includes step to generate it by the users based on the template itself.

Reply:
In that case, we do not want to ignore the `.gitignore` file itself. It is supposed to be checked in.

I don't see why we need to add the claim file to the `.gitignore` file either. I think what you are looking for is `--skip-worktree`.  We want to keep the ignore file for files we actually want to ignore when working with the repository. These files might be TF module cache, IDE specific directories, etc.

Reply:
Thanks, changes to .gitignore reverted